examples/custom-receiver/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "rootDir": "src",
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
When this is set to `true`, build errors relating to other packages do **not** appear. [Reference](https://www.typescriptlang.org/tsconfig/#skipLibCheck) 📚

That includes what was surfaced in #2409 👾

Reply:
Nice thanks 💯  it makes sense to set this as `false`, allows the CI to validate against the packages used by the project